tests/EEGPSR_appendix.tex

```diff
@@ -0,0 +1,144 @@
+\chapter[EEGPSR in detail]{E\textsuperscript{2}GPSR in detail.}
+\label{chap:eegpsr-appendix}
+
+The following section presents detailed examples and descriptions of the skills to be evaluated during the \textit{EEGPSR} test (see \refsec{sec:eegpsr}). Keep in mind that this is provided only as a guideline to illustrate what teams can expect from the test, but it sets no constrains on how the test will be addressed.
+
+\section{Ability examples}
+\begin{itemize}
+	\item \textbf{Advanced manipulation:} The ability of manipulating objects with \textit{two hands} or that require precision and coordination.
+	\begin{itemize}
+		\item Open a bottle (twist, uncap, etc.).
+		\item Pouring cereal in a bowl.
+		\item Placing objects inside a box
+		\item Transporting a tray.
+	\end{itemize}
+
+	\item \textbf{Activity detection:} The ability of detecting human activities is evaluated. It may be requested on purpose such as when the robot is commanded to awake someone, or be triggered by an unexpected event like when a person faints.
+	\begin{itemize}
+		\item A dormant person wakes up.
+		\item People is dancing, reading a book.
```

Comment:
Indeed, I added it as an example, will remove it.

To be honest, I don't find "falling" as an activity. Activities are more like walking, running, singing, dancing, reading, watching tv...